lib/templates/v_0.6.0/api/server-models.js

```diff
@@ -20,7 +20,8 @@ var mongoose = require('mongoose')
 var __name__Schema = mongoose.Schema({
   created:                  { type: Date, default: Date.now }
   , updated:                { type: Date, default: Date.now }
-  , title:                  { type: String, required: '{PATH} is required!' }
```

Comment:
smart.  this causes problems all over the place

i feel like we change it to "name" all the time too.  May want to reconsider using 'name' as default